README.md

```diff
@@ -77,3 +83,10 @@ $ mkdir src
 
 And add the `build` directory to your `.gitignore` file so Gleam's build
 artefacts are not included in your project.
+
+To test if gleam runs you need to add an entry point module.
```

Comment:
This last step isn't needed, let's remove it.

When we have a guide to using Gleam from Elixir we can link to it here.